### PR TITLE
More meeting composite UI tests

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
UI tests in meeting composite for:
* Pre join screen
* Gallery screen
* Chat pane screen (with messages sent and received)
* People pane screen

# Why
For regression testing

# How Tested
Ran locally. Will update snapshots in PR with `ui change` label

### CODE REVIEW

File: packages/react-composites/src/composites/MeetingComposite/SidePane.tsx [hunk 1]
```diff
@@ -105,7 +111,12 @@ export const EmbeddedChatPane = (props: {
   onClose: () => void;
 }): JSX.Element => {
   return (
-    <SidePane hidden={props.hidden} headingText={'Chat'} onClose={props.onClose}>
+    <SidePane
+      hidden={props.hidden}
+      headingText={'Chat'}
+      onClose={props.onClose}
+      dataUiId={'meeting-composite-chat-pane'}
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/MeetingComposite/SidePane.tsx:118]: move to useIdentifiers

File: packages/react-composites/src/composites/MeetingComposite/SidePane.tsx [hunk 2]
```diff
@@ -86,7 +87,12 @@ export const EmbeddedPeoplePane = (props: {
   }, [participantListDefaultProps, callAdapter, chatAdapter]);
 
   return (
-    <SidePane hidden={props.hidden} headingText={'People'} onClose={props.onClose}>
+    <SidePane
+      hidden={props.hidden}
+      headingText={'People'}
+      onClose={props.onClose}
+      dataUiId={'meeting-composite-people-pane'}
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/MeetingComposite/SidePane.tsx:94]: move to useIdentifiers

File: packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx [hunk 3]
```diff
@@ -27,8 +27,18 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
         </CallAdapterProvider>
       </Stack.Item>
       <Stack.Item>
-        <ChatButton checked={props.chatButtonChecked} showLabel={true} onClick={props.onChatButtonClicked} />
-        <PeopleButton checked={props.peopleButtonChecked} showLabel={true} onClick={props.onPeopleButtonClicked} />
+        <ChatButton
+          checked={props.chatButtonChecked}
+          showLabel={true}
+          onClick={props.onChatButtonClicked}
+          data-ui-id="meeting-composite-chat-button"
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx:34]: move to useIdentifiers

File: packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx [hunk 4]
```diff
@@ -27,8 +27,18 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
...
+        />
+        <PeopleButton
+          checked={props.peopleButtonChecked}
+          showLabel={true}
+          onClick={props.onPeopleButtonClicked}
+          data-ui-id="meeting-composite-people-button"
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx:40]: move to useIdentifiers

**Review by alcail — APPROVED** [review]

**Comment by anjulgarg** [comment]
packages/react-composites/tests/browser/call/CallComposite.test.ts-snapshots/page-0-video-gallery-chromium.png
Seems to be an incorrect snapshot.

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**